scripts/ccpp_prebuild.py

```diff
@@ -119,6 +122,33 @@ def setup_logging(debug):
         logging.info('Logging level set to INFO')
     return success
 
+def clean_files(config, static):
+    """Clean files created by ccpp_prebuild.py"""
+    success = True
+    logging.info('Performing clean ....')
+    # Create list of files to remove, use wildcards where necessary
+    files_to_remove = [
+        config['schemes_makefile'],
+        config['schemes_cmakefile'],
+        config['caps_makefile'],
+        config['caps_cmakefile'],
+        config['html_vartable_file'],
+        config['latex_vartable_file'],
+        ]
+    if static:
+        files_to_remove.append(os.path.join(config['caps_dir'], 'ccpp_*_cap.F90'))
+        files_to_remove.append(os.path.join(config['static_api_dir'], '{api}.F90'.format(api=CCPP_STATIC_API_MODULE)))
+    else:
+        files_to_remove.append(os.path.join(config['caps_dir'], '*_cap.F90'))
+        for target_file in config['target_files']:
+            target_file_path = os.path.split(target_file)[0]
+            files_to_remove.append(os.path.join(target_file_path, config['module_include_file'].format(set='*')))
+            files_to_remove.append(os.path.join(target_file_path, config['fields_include_file'].format(set='*')))
+    # Not very pythonic, but the easiest way w/o importing another Python module
+    cmd = 'rm -vf {0}'.format(' '.join(files_to_remove))
```

Comment:
But you already import `os`, why not just call `os.remove`?

Reply:
`os.remove` only works if the file is present (i.e. need to be able to test for it), and doesn't deal with wildcards as far as I know. Happy to be corrected here.

Reply:
True about the file present, or file permission denied issues. My version traps `OSError` so that you get an explanation of what went wrong and an overall failure message.
To handle wildcards, you would have to `import glob`.
In any case, these are minor concerns and what you choose to do is entirely up to the motivation of why you added this feature (the first bullet in https://github.com/NCAR/NEMSfv3gfs/pull/158). If you meet that need, I am okay with the current code. If you need error handling, I would go to the python solution or use a subprocess method that returns an error code.

Reply:
Yes, this solution works for NEMSfv3gfs. In any case, having '-f' as argument to 'rm suppresses any error and always returns a zero exit code (I believe). Otherwise I'd be using subprocess for sure.